docker/conf/homeserver.yaml

```diff
@@ -40,7 +40,7 @@ listeners:
         compress: false
   {% endif %}
 
-  - port: 8008
+  - port: {{ SYNAPSE_PORT or 8008 }}
```

Comment:
Why are we making this configurable?

Reply:
So that the reverse proxy can listen on 8008 and route traffic instead. Complement will only contact the homeserver on 8008/8448 (and making it configurable on Complement's end has been rejected).